src/application/job_manager.py

```
@@ -0,0 +1,41 @@
+from typing import List, Optional, Dict, Any
```

Ruff (0.12.2)

`typing.List` is deprecated, use `list` instead

(UP035)